.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,116 @@
+        type: boolean
+      environment:
+        description: 'Environment to run tests against'
+        type: environment
+        required: true
+  push:
+    branches: ['terra-application-v1']
+  pull_request:
+    branches: ['terra-application-v1']
+  schedule:
+    - cron: '0 1 * * SUN'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v3
+      - name: Setup Project
+        uses: ./.github/actions/setup
+      - if: github.event_name == 'pull_request'
+        name: Danger
+        uses: danger/danger-js@9.1.8
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # https://github.com/danger/danger-js/issues/557#issuecomment-664851950
+          DANGER_DISABLE_TRANSPILATION: true
+      - name: Linting Project
+        run: |
+          npm run lint
+      - name: Testing Project
+        run: |
+          npm run jest
+  wdio:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        theme: [default, lowlight, fusion]
+      
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v3
+      - name: Setup Project
+        uses: ./.github/actions/setup      
+      - name: Compile Project Default
+        run: |
+          npm run compile:prod -- --output-path build/default
+        if: matrix.theme == 'default'
+      - name: Compile Project Lowlight
+        run: |
+          npm run compile:lowlight -- --output-path build/lowlight
+        if: matrix.theme == 'lowlight'
+      - name: Compile Project Fusion
+        run: |
+          npm run compile:fusion -- --output-path build/fusion
+        if: matrix.theme == 'fusion'
+      - name: Run WDIO Tests for theme ${{ matrix.theme }}
+        env:
+          SITE: build/${{ matrix.theme }}
+        run: npm run wdio-${{ matrix.theme }}
+
+  release-and-deploy:
+    runs-on: ubuntu-latest
+
+    needs: [build, wdio]
+    if: github.ref == 'refs/heads/terra-application-v1'
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v2
+      - name: Setup GitHub Config
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+      - name: Setup Project
+        uses: ./.github/actions/setup
+      - name: Build
+        env:
+          TERRA_DEV_SITE_NEW_RELIC_LICENSE_KEY: ${{ secrets.TERRA_DEV_SITE_NEW_RELIC_LICENSE_KEY }}
+          TERRA_DEV_SITE_NEW_RELIC_APPLICATION_ID: "141260567"
+          TERRA_DEV_SITE_PUBLIC_PATH: /terra-application/
+        run: npm run compile:prod
+      - name: Release
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: npx terra release
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./build
```

Comment:
How are the deployments being managed between different versions of terra-application currently? Wouldn't running from v1 override the deployment from main for example?

Reply:
There isn't any existing deployment for v1, hence this PR. I'm not sure if the v1 branch will override the deployment from main. Is there anything I need to do to make sure that doesn't happen?

Reply:
Ok, I wasn't sure and checked. Looks like both branches will be deploying to the same branch: https://github.com/cerner/terra-application/commits/gh-pages 
Your deployment commit from v1 branch here: https://github.com/cerner/terra-application/actions/runs/5928158603/job/16073157681?pr=344 
Not sure if that is ok or not. 
If that is not Ok, we can just delete or skip the deploy part.

Reply:
I'll delete it for now so it doesn't overwrite main. We can do some fancy setup later so that we can have 2 setup sites.

Reply:
Removed deployment here: 1e90b35